ansible/tasks/test-image.yml

```diff
@@ -52,11 +52,11 @@
   when: ebssurrogate_mode
 
 - name: Drop pgTap extension
-  shell: /usr/lib/postgresql/bin/psql -U postgres -h localhost -d postgres -c "DROP extension if exists pgtap";
+  shell: /usr/lib/postgresql/bin/psql -U supabase_admin -h localhost -d postgres -c "DROP extension if exists pgtap";
   when: ebssurrogate_mode
 
 - name: Drop extension test function
-  shell: /usr/lib/postgresql/bin/psql -U postgres -h localhost -d postgres -c "DROP FUNCTION IF EXISTS install_available_extensions_and_test";
+  shell: /usr/lib/postgresql/bin/psql -U supabase_admin -h localhost -d postgres -c "DROP FUNCTION IF EXISTS install_available_extensions_and_test";
```

Comment:
Also on closer look, this function is created by [unit-tests](https://github.com/supabase/postgres/pull/550/files#diff-2d01095c3e13c0a570e306aa7f34fbb31b8e1e1612708dceb3b0b752628fe89bR34) which is run as postgres user. so this change is unnecessary